### PR TITLE
Rework moderators activity query

### PR DESCRIPTION
**Pull request by scossar**
Any suggestions for improving this would be great. 

I can't find a way of returning the `avatar_template` other than getting it in the `DB.query(query).each` block.

### CODE REVIEW

**Comment by jjaffeux** [comment]
@scossar have you looked how I do it in `report_post_edits` ?

**Comment by scossar** [comment]
Yes, I think I'm doing it the same way here:

` mod[:user_avatar_template] = User.avatar_template(row.username, row.uploaded_avatar_id)`

I was trying to find a way to get the avatar template from the DB query to avoid having to look it up in the `DB.query(query).each` loop, but that doesn't seem possible.

**Comment by SamSaffron** [comment]
hmmm when is this ready to merge?

File: app/models/report.rb [hunk 1]
```diff
@@ -722,152 +722,144 @@ def self.report_moderators_activity(report)
...
+    ORDER BY m.username
+    SQL
 
-    DB.query(pm_count_query).each do |row|
-      mod_data[row.user_id][:pm_count] = row.pm_count
+    DB.query(query).each do |row|
+      mod = {}
+      mod[:username] = row.username
+      mod[:user_id] = row.user_id
+      mod[:user_avatar_template] = User.avatar_template(row.username, row.uploaded_avatar_id)
```
Comment (SamSaffron) [inline app/models/report.rb]: Yeah this is how you get templates, note no query runs here ...

**Comment by scossar** [comment]
> hmmm when is this ready to merge?

I'm happy with it.

**Comment by SamSaffron** [comment]
OK, let's try this out then

**Pull request merged**